react/README.md

```diff
@@ -0,0 +1,27 @@
+# reactapp
```

Comment:
remove

Reply:
Should  be resolved in https://github.com/radiasoft/sirepo/pull/4667/commits/9cc4554f84321a4ebad33583e8e797f7fec20a7d